packages/services/team/worker/src/workers/member_remove.rs

```diff
@@ -28,10 +28,13 @@ async fn worker(ctx: &OperationContext<team::msg::member_remove::Message>) -> Gl
 			)
 		},
 		async {
-			Ok(msg!([ctx] user::msg::update(user_id) {
-				user_id: Some(user_id.into()),
-			})
-			.await?)
+			Ok(
+				// TODO: Dispatch directly from ctx
```

Comment:
dedicated signal for JoinedTeam or something, then the wf can decide what to do with it (completely remove UserUpdateMsgDispatch)